src/test/data/JsonUserPrefsStorageTest/ExtraValuesUserPref.json

```diff
@@ -9,5 +9,5 @@
       "z" : 99
     }
   },
-  "addressBookFilePath" : "addressbook.json"
+  "addressBookFilePath" : "resume.json"
```

Comment:
is it possible to change the addressBookFilePath to resumeBookFilePath or smt?

Reply:
Yes I will make that change